include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -3635,11 +3635,11 @@ namespace internal
       }
 
       // sort for support points
-      std::sort(sp_cell_dofs.begin(),
-                sp_cell_dofs.end(),
-                [](const auto &a, const auto &b) {
-                  return std::get<0>(a) < std::get<0>(b);
-                });
+      std::stable_sort(sp_cell_dofs.begin(),
+                       sp_cell_dofs.end(),
+                       [](const auto &a, const auto &b) {
+                         return std::get<0>(a) < std::get<0>(b);
+                       });
```

Comment:
What's the benefit of stable sort over sort in this particular case?

Reply:
This is not needed right now; only once we want to support multiple components and we need to guarantee some ordering. I changed it here so that we don't forget it later.

Reply:
Nice, I didn't think about this :)